dll/win32/imm32/imm.c

```diff
@@ -61,27 +62,36 @@ SHAREDINFO g_SharedInfo = { NULL };
 BYTE g_bClientRegd = FALSE;
 HANDLE g_hImm32Heap = NULL;
 
-static PWND FASTCALL ValidateHwndNoErr(HWND hwnd)
+static PWND FASTCALL ValidateHwndNoErr(HWND hWnd)
 {
     PCLIENTINFO ClientInfo = GetWin32ClientInfo();
     INT index;
     PUSER_HANDLE_TABLE ht;
     WORD generation;
 
     /* See if the window is cached */
-    if (hwnd == ClientInfo->CallbackWnd.hWnd)
+    if (hWnd == ClientInfo->CallbackWnd.hWnd)
         return ClientInfo->CallbackWnd.pWnd;
 
-    if (!NtUserValidateHandleSecure(hwnd))
-        return NULL;
-
     ht = g_SharedInfo.aheList; /* handle table */
```

Comment:
Please use `SharedPtrToUser(gSharedInfo.aheList)`

Reply:
`SharedPtrToUser` is not available from `IMM32`.